codegenerator/cli/npm/envio/src/Env.res

```diff
@@ -15,7 +15,6 @@
 // resets the timestampCaughtUpToHeadOrEndblock after a restart when true
 let updateSyncTimeOnRestart =
   envSafe->EnvSafe.get("UPDATE_SYNC_TIME_ON_RESTART", S.bool, ~fallback=true)
-let batchSize = envSafe->EnvSafe.get("MAX_BATCH_SIZE", S.option(S.int))
```

Comment:
Is this no longer configurable?

Reply:
Via config.yaml

Reply:
Hmmm, I'm not sure about that because then you can't change it in production

Reply:
@DZakh can we keep this?

I feel like its important if for example you suddenly find you're running out of memory or something

Reply:
Theoretically possible with changing config, and then running `pnpm codegen` && `pnpm start`